esp/src/src/nls/es/hpcc.ts

```diff
@@ -145,19 +146,22 @@ export = {
     ConfigureService: "Configurar servicio",
     ConfirmPassword: "Confirme la Contraseña",
     ConfirmRemoval: "¿Seguro que quieres hacer esto?",
+    ConnectionID: "ID de conexión",
```

Comment:
As long as we're consistently using ID instead of identidad, this should be ok, otherwise I would use the proper term for identification

Reply:
We use both versions, but ID is used more across the platform